lutris/gui/widgets/game_bar.py

```diff
@@ -269,7 +276,10 @@ def on_install_clicked(self, button):
         """Handler for installing service games"""
         self.service.install(self.db_game)
 
-    def on_game_state_changed(self, game):
+    def on_game_launcher_state_changed(self, game_launcher: GameLauncher):
```

Comment:
Are we going to change the signal and handler names to say 'launcher'? If so, you missed a bunch.